dev-libs/libnl/files/libnl-3.11-skip-private-netns.patch

```diff
@@ -0,0 +1,158 @@
+diff --git a/tests/cksuite-all-netns.c b/tests/cksuite-all-netns.c
```

Comment:
This looks about right but needs rationale and the usual patch headers and information.